src/main/java/stringaddcalculator/domain/ValidationNumber.java

```diff
@@ -0,0 +1,25 @@
+package stringaddcalculator.domain;
+
+public class ValidationNumber {
+
+    private static final String NUMERICAL_REGULAR_EXPRESSION = "^[0-9]+$]";
+
+    public static boolean isBlank(String expression) {
+        if (expression == null || expression.trim().isEmpty()) {
+            return true;
+        }
+        return false;
```

Comment:
아래와 같이도 사용할 수 있지 않을까요?

```suggestion
        return expression == null || expression.trim().isEmpty();
```

Reply:
이 부분은 충분히 바로 리턴할 법한데,
생각을 적게하고 작성한것 같네요.. !
반영했습니다 !